src/Register.c

```diff
@@ -202,7 +202,7 @@ ZydisRegister ZydisRegisterGetLargestEnclosing(ZydisMachineMode mode, ZydisRegis
         return static_reg;
     }
 
-    static const ZyanU8 GPR8_MAPPING[20] =
+    static const ZyanU8 GPR8_MAPPING[36] =
```

Comment:
We should change semantics of all `Register*` APIs based on a `ZyanBool apx` parameter.